docs/apache-airflow/index.rst

```diff
@@ -15,65 +15,120 @@
     specific language governing permissions and limitations
     under the License.
 
+What is Airflow?
+=========================================
 
+`Apache Airflow <https://github.com/apache/airflow>`_ is an open-source platform for developing, scheduling,
```

Comment:
Ilove the summary,  But I think we should mention the "orchestration" here. This is a well know subject. Also I would not be shy - I think we should mention that Airlfow is "As of 2022 when we write this, Airflow is the most popular orchestration platform for batch oriented workflows". I do believe it's truth. I have no hard data to back it up though. 

I think it can have really good "impact" on choosing Airflowl for someone to reach the page.

Reply:
I think adding orchestration here is probably a good idea, but I say we defer it to another PR as we want to match the wording on the homepage as well.